events-backend/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
```

Comment:
This file is needed because of the way our modules are deployed via nest into Wildfly (on the main Hawkular build). On our own dist, we don't need this file.